angular/lib/src/components/select/select.component.ts

```diff
@@ -75,14 +76,16 @@ export class SelectItemDirective<Item> {
 			</div>
 			@if (state.open && state.visibleItems.length) {
 				<ul
+					[id]="state.id + '-menu'"
 					[auUse]="menuDirective"
 					[class]="'dropdown-menu show ' + (menuClassName || '')"
 					[attr.data-popper-placement]="state.placement"
 					(mousedown)="$event.preventDefault()"
 				>
 					@for (itemContext of state.visibleItems; track itemCtxTrackBy($index, itemContext)) {
 						<li
-							[class]="'au-select-item dropdown-item position-relative' + (itemContext === state.highlighted ? ' bg-primary text-light' : '')"
+							[class]="'au-select-item dropdown-item position-relative'"
+							[class.bg-text-bg-primary]="itemContext === state.highlighted"
```

Comment:
is this this good in dark mode ?

Reply:
I don't know what's happened, but the point is that the bg-text-bg-primary class is not included anymore :(

And it should be fine for the dark mode, as it's like this Bootstrap manages it (there is no color switch in bootstrap. For example https://getbootstrap.com/docs/5.3/components/toasts/#color-schemes).

@ExFlo, how is it supposed to be done ?

Reply:
Oh, my bad, it's `text-bg-primary` that needs to be used !